Documentation/project-docs/developer-guide.md

```diff
@@ -38,21 +37,38 @@ the managed build which produces the MSIL code and nuget packages that make up C
 Calling the script `build` attempts to build both the native and managed code.
 Only use it when the parameters that you are passing to the script apply for both components. Otherwise, use the scripts `build-native` and `build-managed` respectively.
 
-**Examples**
+Most of the build configurations are defaulted based on where you are building but here are a few common parameters to customize the build configuration:
 
-- Building in debug mode for platform x64
+- `-framework` identifies the target framework for the build. It defaults to `netcoreapp` but possible values include `netcoreapp`, `netfx` or `uap`. (msbuild property `TargetGroup`)
```

Comment:
That is a good point and I was thinking about that but decided against it because I wanted these to be the short-hand for these properties. I agree that remembering the msbuild properties is a pain but my hope is we will soon be able to use the dev-workflow scripts for individual products (@mattgal is working on that) and at which point the short-hand can be used everywhere. Also if you use the help `build -?` it will show you what the msbuild properties are for the given shortcuts.